charts/csi-baremetal-driver/templates/controller.yaml

```diff
@@ -16,6 +16,7 @@ spec:
         app.kubernetes.io/name: csi-baremetal
         app: csi-baremetal-controller
         role: csi-do
+        release: {{ .Release.Name }}
```

Comment:
Please put comments why this option is present in charts and make corresponding PR to operator.

Reply:
comment in every chart?

Reply:
These charts will be deprecated soon. Please add comment to the chart in Operator repo.